### PR TITLE
Reorganise the render/ package

### PR DESCRIPTION
**Pull request by tomwilkie**
Aim here is for no code changes, but to break down mapping.go and topology.go into separate files for each topology, which container both the mapping functions and the topology renderers.

### CODE REVIEW

File: test/fixture/report_fixture.go [hunk 1]
```diff
@@ -381,6 +382,7 @@ var (
 						kubernetes.PodState:        "running",
 						kubernetes.PodContainerIDs: ServerContainerID,
 						kubernetes.ServiceIDs:      ServiceID,
+						report.HostNodeID:          ServerHostNodeID,
```
Comment (paulbellamy) [inline test/fixture/report_fixture.go]: Why did these have to change?

**Comment by paulbellamy** [comment]
The files in this, seem structured to match the probes (docker, kubernetes, process). Thoughts about having one-file-per-target? i.e. making them match `app/api_topologies.go` instead?

File: test/fixture/report_fixture.go (hunk 1 above)
Comment (tomwilkie) [inline test/fixture/report_fixture.go]: Oh it didn't; I added it for completeness (all the other nodes have the host ID).

**Comment by paulbellamy** [comment]
Discussed elsewhere, to rename the files according to "top-level" renderer (i.e. "process", "container", "pod", and "host"), then each file includes any sub-views (so ContainerHostname view goes in "container").

**Comment by tomwilkie** [comment]
@paulbellamy PTAL.

**Pull request merged**